packages/fxa-settings/src/components/Settings/SubRow/index.tsx

```diff
@@ -48,47 +69,103 @@ const SubRow = ({
     </span>
   );
 
+  const ExtraInfoLink = () => {
+    return linkExternalProps ? (
+      <LinkExternal
+        href={linkExternalProps.href}
+        className="link-blue inline ms-2"
+        data-glean-id={linkExternalProps.gleanDataAttrs?.id}
+      >
+        {linkExternalProps.children}
+      </LinkExternal>
+    ) : null;
+  };
+
   return (
     <div
       className={classNames(
-        'flex flex-col @mobileLandscape/unitRow:flex-row w-full max-w-full mt-8 p-2 @mobileLandscape/unitRow:mt-4 @mobileLandscape/unitRow:rounded-lg border items-start text-sm gap-4',
+        'flex flex-col w-full max-w-full mt-8 p-4 @mobileLandscape/unitRow:mt-4 @mobileLandscape/unitRow:rounded-lg border items-start text-sm gap-2',
         {
           'bg-grey-10 border-transparent': !isEnabled,
           'bg-white border-grey-100': isEnabled,
-          '@mobileLandscape/unitRow:items-center': !showDescription,
         }
       )}
-      data-testid={`${idPrefix}-sub-row`}
     >
-      <div className="flex flex-row justify-between flex-1 flex-wrap items-center">
-        <div className="flex items-center gap-2">
-          <div className="grow-0 shrink-0">{icon}</div>
-          <p className="font-semibold">{title}</p>
-        </div>
-        <div className="flex items-center gap-1">
-          <StatusIcon />
-          {message}
+      <div
+        className={classNames(
+          'flex flex-col @mobileLandscape/unitRow:flex-row w-full max-w-full items-start text-sm gap-4'
+        )}
+        data-testid={`${idPrefix}-sub-row`}
+      >
+        <div className="flex flex-row justify-between flex-1 flex-wrap items-center">
+          <div className="flex items-center gap-2">
+            <div className="grow-0 shrink-0">{icon}</div>
+            <p className="font-semibold">{localizedRowTitle}</p>
+          </div>
+          <div>
+            <div className="flex items-center gap-1">
+              <StatusIcon />
+              {message}
+            </div>
+            {!localizedDescription &&
+              !localizedInfoMessage &&
+              linkExternalProps && <ExtraInfoLink />}
+          </div>
+          {localizedDescription && (
+            <p className="text-xs w-full mx-2 mt-2">
```

Comment:
I know it makes sense to keep using `text-xs` until we update the font-size everywhere but it really is so small when there's a decent amount of copy 😅

Reply:
You're right, it is very small - maybe best to switch to `text-sm`